staging/src/k8s.io/code-generator/cmd/import-boss/main.go

```diff
@@ -81,6 +81,30 @@ func main() {
 		"k8s.io/kubernetes/pkg/...",
 		"k8s.io/kubernetes/cmd/...",
 		"k8s.io/kubernetes/plugin/...",
+		"./vendor/k8s.io/api/...",
+		"./vendor/k8s.io/apiextensions-apiserver/pkg/...",
+		"./vendor/k8s.io/apimachinery/...",
+		"./vendor/k8s.io/apiserver/...",
+		"./vendor/k8s.io/cli-runtime/...",
+		"./vendor/k8s.io/client-go/...",
+		"./vendor/k8s.io/cloud-provider/...",
+		"./vendor/k8s.io/cluster-bootstrap/...",
+		"./vendor/k8s.io/code-generator/...",
+		"./vendor/k8s.io/component-base/...",
+		"./vendor/k8s.io/cri-api/...",
+		"./vendor/k8s.io/csi-translation-lib/...",
+		"./vendor/k8s.io/kube-aggregator/...",
+		"./vendor/k8s.io/kube-controller-manager/...",
+		"./vendor/k8s.io/kubectl/...",
+		"./vendor/k8s.io/kubelet/...",
+		"./vendor/k8s.io/kube-proxy/...",
+		"./vendor/k8s.io/kube-scheduler/...",
+		"./vendor/k8s.io/legacy-cloud-providers/...",
+		"./vendor/k8s.io/metrics/...",
+		"./vendor/k8s.io/node-api/...",
+		"./vendor/k8s.io/sample-apiserver/...",
+		"./vendor/k8s.io/sample-cli-plugin/...",
+		"./vendor/k8s.io/sample-controller/...",
```

Comment:
this list is misplaced here in the file. Even the three pre-existing packages above are wrong. We have to move the list where we invoke importboss (some hack/ script?).

And there iterate over `staging/src/k8s.io/*` and generate this list automatically.

Reply:
Done, now generating this list in [hack/verify-import-boss.sh](https://github.com/kubernetes/kubernetes/pull/83526/files#diff-d19bc83764e63f6805cab0f1ce53ea92R28)